<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationDialog</class>
 <widget class="QDialog" name="CalibrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calibration Measurements</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Measurements</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Create default measurements for:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1 Port SOL calibration</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2 Port SOLT calibration</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3 Port SOLT calibration</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4 Port SOLT calibration</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableWidget" name="table"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="bAdd">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Add</string>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
            <property name="icon">
             <iconset theme="list-add" resource="../icons.qrc">
              <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bDelete">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Delete</string>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
            <property name="icon">
             <iconset theme="list-remove" resource="../icons.qrc">
              <normaloff>:/icons/remove.png</normaloff>:/icons/remove.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bMoveUp">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Move up</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="go-up" resource="../icons.qrc">
              <normaloff>:/icons/up.png</normaloff>:/icons/up.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bMoveDown">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Move down</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="go-down" resource="../icons.qrc">
              <normaloff>:/icons/down.png</normaloff>:/icons/down.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="measure">
            <property name="text">
             <string>Measure</string>
            </property>
            <property name="icon">
             <iconset theme="media-playback-start" resource="../icons.qrc">
              <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearMeasurement">
            <property name="text">
             <string>Clear</string>
            </property>
            <property name="icon">
             <iconset theme="edit-delete" resource="../icons.qrc">
              <normaloff>:/icons/trash.png</normaloff>:/icons/trash.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>18</width>
              <height>186</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CalibrationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CalibrationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
